recipes/xapian-core/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "xapian-core" %}
+{% set version = "1.4.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://oligarchy.co.uk/xapian/{{ version }}/xapian-core-{{ version }}.tar.xz
+  sha256: 13f08a0b649c7afa804fa0e85678d693fd6069dd394c9b9e7d41973d74a3b5d3
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: |
+    LDFLAGS=$LDFLAGS \
+    ./configure \
+    CXXFLAGS=-I$PREFIX/include \
+    --prefix=$PREFIX
+
+    make && make install
+    rm -rf $PREFIX/share/doc/xapian-core
+
+  run_exports:
+    {{ pin_subpackage('xapian-core', max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+  host:
+    - zlib
```

Comment:
If xapian is linking  against zlib, zlib should also be in `reqs/run`

Reply:
Here I followed @loriab's [suggestion](https://github.com/conda-forge/staged-recipes/pull/6495#discussion_r210112335) and removed it, since [zlib defines `run_exports`](https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L14-L17).

If you prefer "explicit is better than implicit" I can add it back, no problem.

Reply:
Oh right, sorry!  I am still getting used to run_exports